specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4519,9 +4519,9 @@
         "@odata.type": {
           "type": "string"
         },
-		"name": {
+        "name": {
           "type": "string",
-          "description": "The name of the skill which uniquely identifies the skill within the skillset. Skill names must be unique per skillset."
+          "description": "The name of the skill which uniquely identifies the skill within the skillset. Skill names must be unique per skillset. Skill's with no skillname defined will be given a default name of its index in the skillset array."
```

Comment:
- The first two sentences seem to me to say the same thing. Maybe get rid of the second one?
- Skill's -> Skills (no apostrophe since it's not possessive)
- skillname -> name